hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/SCMContainerManager.java

```diff
@@ -215,23 +222,11 @@ public ContainerInfo allocateContainer(final ReplicationType type,
       throws IOException {
     lock.lock();
     try {
-      final ContainerInfo containerInfo; containerInfo = containerStateManager
+      final ContainerInfo containerInfo;
```

Comment:
Can be put as single statement --> 
final ContainerInfo containerInfo = containerStateManager.allocateContainer(pipelineManager, type, replicationFactor, owner);

Reply:
Done